libraries/botbuilder-dialogs-adaptive/src/actions/logAction.ts

```diff
@@ -55,7 +55,7 @@ export class LogAction<O extends object = {}> extends Dialog<O> {
 
     /**
      * Starts a new dialog and pushes it onto the dialog stack.
```

Comment:
Should we use the Dialog class reference here?, more lines with this point on this file.

Reply:
I didn't link to `Dialog` because the class itself implements it. Also, it's not linked in their C# counterparts either.

Reply:
Solved!